target/linux/ath79/dts/ar7100_mikrotik_routerboard-4xx.dtsi

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: GPL-1.0-or-later OR MIT
+
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar7100.dtsi"
+
+/ {
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	aliases {
+		led-boot = &led_user;
+		led-failsafe = &led_user;
+		led-running = &led_user;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_user: user {
+			label = "mikrotik:yellow:user";
+			gpios = <&gpio 4 GPIO_ACTIVE_HIGH>;
+			default-state = "on";
+		};
+
+		led_1: led1 {
```

Comment:
I don't think you need these DT labels (e.g. led_1) except for led_user. Remove them.

Reply:
Deleted